application/src/Api/Representation/AbstractResourceEntityRepresentation.php

```diff
@@ -277,7 +277,7 @@ public function values()
      *   by default.
      * - lang: (null) Get values of this language only. Returns values of all
      *   languages by default.
-     * @return RepresentationInterface|mixed
+     * @return ValueRepresentation[]|mixed
```

Comment:
I guess this should really be `ValueRepresentation|ValueRepresentation[]|mixed`. Maybe we should just give up and specify it as `mixed`. If we're going to specify though, a lone `ValueRepresentation` is the "usual" case so should probably be first.

Reply:
When there are multiple possible outputs, it's better to use the most frequent one, because this is the one that is used by the editor, and `mixed` is not helpy in that case, so I prefer to use the three possible values here. Fixed in next commit.